docs/src/content/docs/guides/project-structure.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Project Structure
+description: Learn how to organize files in your Starlight project.
+---
+
+This guide will show you how a Starlight project is organized and what different files in your project do. Starlight projects generally follow the same file and directory structure as other Astro projects. See [Astro’s project structure documentation](https://docs.astro.build/en/core-concepts/project-structure/) for more detail.
```

Comment:
```suggestion
This guide will show you how a Starlight project is organized and what the different files in your project do.

Starlight projects generally follow the same file and directory structure as other Astro projects. See [Astro’s project structure documentation](https://docs.astro.build/en/core-concepts/project-structure/) for more detail.
```
Nit: I like adding "the", and I like separating these concepts into their own paragraphs.